apps/spruce/src/components/Redirects/PatchRedirect.tsx

```diff
@@ -1,7 +1,12 @@
 import { useParams, Navigate } from "react-router-dom";
-import { getVersionRoute, slugs } from "constants/routes";
+import { getPatchRoute, slugs } from "constants/routes";
 
 export const PatchRedirect: React.FC = () => {
   const { [slugs.versionId]: versionId } = useParams();
-  return <Navigate replace to={getVersionRoute(versionId ?? "")} />;
+  return (
+    <Navigate
+      replace
+      to={getPatchRoute(versionId ?? "", { configure: true })}
+    />
+  );
```

Comment:
In DEVPROD-5907 I updated this component so that navigating to `/patch/<id>` for an activated task would redirect to `/version/<id>`. Now that directs to `/patch/<id>/configure` which seems less desirable; could we perhaps still direct to the version page if the patch is activated?

Reply:
what do you think about the logic now 👀